### PR TITLE
add decorator position blurb

### PR DESCRIPTION
**Pull request by NullVoxPopuli**

### CODE REVIEW

File: source/2020-03-27-the-ember-times-issue-141.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: The Ember Times - Issue No. 141
-author: the crowd
+author: the crowd, Preston Sego
```
Comment (MinThaMie) [inline source/2020-03-27-the-ember-times-issue-141.md]:
```suggestion
author: Preston Sego, the crowd
```

File: source/2020-03-27-the-ember-times-issue-141.md [hunk 2]
```diff
@@ -143,4 +141,4 @@ That's another wrap! ✨
 
 Be kind,
 
-the crowd and the Learning Team
+the crowd, Preston Sego and the Learning Team
```
Comment (MinThaMie) [inline source/2020-03-27-the-ember-times-issue-141.md]:
```suggestion
Preston Sego, the crowd and the Learning Team
```

**Review by MinThaMie — APPROVED** [review]

File: source/2020-03-27-the-ember-times-issue-141.md (hunk 2 above)
Comment (NullVoxPopuli) [reply source/2020-03-27-the-ember-times-issue-141.md]: obligatory oxford comma question?

**Comment by ijlee2** [comment]
I'll go ahead and merge this. Thanks for your help, Anne and Preston!

**Pull request merged**